lib/core/db_models/old/job_summary_data_model.dart

```diff
@@ -25,44 +25,46 @@ class JobSummaryDataModel {
     @required Map<String, dynamic> map,
     @required String docID,
   }) {
+    print("data map is ${map}");
+    print("data doc id is ${docID}");
```

Comment:
Print statement should now have line number and file name printed with parameters. Please check coming standards doc for reference